packages/shared-ux/error_boundary/src/ui/message_components.tsx

```diff
@@ -55,7 +55,7 @@ const CodePanel: React.FC<ErrorCalloutProps & { onClose: () => void }> = (props)
         </EuiPanel>
       </EuiFlyoutHeader>
       <EuiFlyoutBody>
-        <EuiCodeBlock>
+        <EuiCodeBlock data-test-subj="errorBoundaryFatalDetailsErrorString">
```

Comment:
I think this is the change that is throwing off error_boundary.test.tsx 
```    (await test.findByTestId('errorBoundaryRecoverablePromptReloadBtn')).click();```
and 
``` (await findByTestId('errorBoundaryFatalPromptReloadBtn')).click(); ``` solved the test failures locally for me!